rstr/xeger.py

```diff
@@ -28,8 +28,8 @@ class Xeger(RstrBase):
     def __init__(
         self, _random: '_Random' = typing.cast('_Random', random), **custom_alphabets: str,
     ) -> None:
-        super(Xeger, self).__init__(_random, **custom_alphabets)
-        self._cache: Dict[str, str] = dict()
```

Comment:
Nitpick, but I kind of prefer `dict()` over `{}` for cases like this where it's an initial/default/fallback value. It doesn't actually matter in this specific instance, but I've often seen bugs where a single dict instance is passed around as a default value, when really it should be a new dict created.